README.md

```diff
@@ -21,6 +21,8 @@ For all the FORKED repos (not the read only repos) you can now make changes dire
 4. Push branch to your fork `git push origin my-fix`.
 5. Create pull request from GitHub.
 
+> Please do not forget to run `composer install` in der root directory of the module and `npm install` in the `/resources` directory of the module to download all needed dependencies.
```

Comment:
@dev7ch do you mean "in the repos directory" or in the root of luya-env-dev? I am not sure if this information is not very useful as it's depending on the task which wants to be done. 

So maybe its better to have an extra section which says something about "how to work with assets (js, css, etc)." in the luya-env-dev context

Reply:
@nadar well, I stumbled into that problem when i started to fix an issue in the luya-module-admin and coz bs4 is loaded from the vendors it is quite useful information imo.

I wrote **root directory of the module**,  but yes might be right it could be done more clear specific and, with  an example maybe.

Reply:
@dev7ch yes i had the same problem, so i think its good to have a separate title just with informations about this problem. As its more important then just a quote.

Reply:
ok, i ll do that.